__test__/Port.test.js

```diff
@@ -0,0 +1,38 @@
+const Port = require('../src/Port.js');
+
+describe('Port', () => {
```

Comment:
Code here is fine, but all the indentation is at 2 spaces and not 4. Would recommend installing prettier and running ti across your whole project to keep things consistent!